storage/queued_country.hpp

```diff
@@ -6,6 +6,9 @@
 namespace storage
 {
 /// Country queued for downloading.
+/// @TODO(bykoianko) This class assume that a map may consist of one or two mwm files.
+/// But single mwm files are used now. So this class should be redisigned to support
```

Comment:
Why not to fix it right now? The fix should be quite simple - QueuedCountry should have only two fields: id of a downloaded country and a boolean flag - was country downloaded or not.